### ISSUE TITLE
[Bug]: Icons in context menu missing

### ISSUE THREAD

**Issue by citronalco** [nextcloud/maps#1019]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Maps app **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

In the conext menu (right click on map) all options ("add favourites",...) are correctly display, but all icons are missing except "report error" and "share with a conversation".

I'm using Apache2 and have installed Nextcloud in the DocumentRoot.

Excerpt of Apache's log file shows:
```
188.194.XXX.XXX - - [24/Mar/2023:14:39:48 +0100] "GET /index.php/svg/core/actions/starred?color=000000 HTTP/1.1" 302 1084 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
188.194.XXX.XXX - - [24/Mar/2023:14:39:49 +0100] "GET /index.php/svg/core/actions/share?color=000000 HTTP/1.1" 302 1084 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
188.194.XXX.XXX - - [24/Mar/2023:14:39:49 +0100] "GET /index.php/svg/core/actions/user?color=000000 HTTP/1.1" 302 1084 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
188.194.XXX.XXX - - [24/Mar/2023:14:39:49 +0100] "GET /index.php/svg/core/filetypes/location?color=00cc00 HTTP/1.1" 302 1084 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
188.194.XXX.XXX - - [24/Mar/2023:14:39:49 +0100] "GET /index.php/svg/core/places/picture?color=000000 HTTP/1.1" 302 1084 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
188.194.XXX.XXX - - [24/Mar/2023:14:39:49 +0100] "GET /index.php/svg/core/filetypes/location?color=0000cc HTTP/1.1" 302 1084 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0"
```
Haven't seen any other suspicious entries in the log.
Opening one of the links while being logged in in Nextcloud leads to index.php

I have seen someone else reported the issue already here: https://help.nextcloud.com/t/some-icons-are-not-found/154598

### Steps to reproduce

1. Do a right click on the map to open the context menu.

### Expected behavior

Context show all icons

### Installation method

Manual installation

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 7.4

### Web server

Apache (supported)

### Database engine version

PostgreSQL

### Is this bug present after an update or on a fresh install of the server?

Fresh Nextcloud Server install

### Is this bug present after an update or on a fresh install of the app?

Fresh Nextcloud Maps install (never installed before)

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [X] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
Enabled:
  - activity: 2.17.0
  - apporder: 0.15.0
  - bookmarks: 12.1.0
  - calendar: 4.3.1
  - circles: 25.0.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - contacts: 5.2.0
  - contactsinteraction: 1.6.0
  - dav: 1.24.0
  - event_update_notification: 2.0.0
  - federatedfilesharing: 1.15.0
  - federation: 1.15.0
  - files: 1.20.1
  - files_external: 1.17.0
  - files_fulltextsearch: 25.0.0
  - files_fulltextsearch_tesseract: 25.0.0
  - files_pdfviewer: 2.6.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - firstrunwizard: 2.14.0
  - fulltextsearch: 25.0.0
  - fulltextsearch_elasticsearch: 25.0.0
  - ldap_write_support: 1.7.0
  - lookup_server_connector: 1.13.0
  - mail: 2.2.4
  - maps: 1.0.0
  - music: 1.8.1
  - notes: 4.7.2
  - notifications: 2.13.1
  - oauth2: 1.13.0
  - photos: 2.0.1
  - previewgenerator: 5.2.1
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - related_resources: 1.0.4
  - richdocuments: 7.1.2
  - settings: 1.7.0
  - spreed: 15.0.4
  - systemtags: 1.15.0
  - tasks: 0.14.5
  - text: 3.6.0
  - theming: 2.0.1
  - twofactor_backupcodes: 1.14.0
  - updatenotification: 1.15.0
  - user_ldap: 1.15.0
  - viewer: 1.9.0
  - weather_status: 1.5.0
  - workflowengine: 2.7.0
Disabled:
  - admin_audit: 1.3.0
  - bruteforcesettings: 2.5.0
  - dashboard: 7.0.0
  - encryption
  - files_downloadactivity: 1.16.0
  - logreader: 2.0.0
  - nextcloud_announcements: 1.0
  - password_policy: 1.1.0
  - recommendations: 0.8.0
  - serverinfo: 1.1.1
  - support: 1.3.0
  - survey_client: 1.1.0
  - suspicious_login
  - twofactor_totp
  - user_status: 1.0.1
```

**Comment by copyrights** [comment]
I'm using it in a container (nginx + nextcloud:fpm) and get HTTP 404 for those icons
```
172.xxx.xxx.xxx - - [23/Apr/2023:20:47:23 +0000] "GET /index.php/svg/core/actions/user?color=000000 HTTP/1.1" 404 5 "https://cloud.domain.tld/index.php/apps/maps/" "Mozilla/5.0 (xxxxxxxxxxxxxxx) Gecko/20100101 Firefox/112.0"
172.xxx.xxx.xxx - - [23/Apr/2023:20:47:23 +0000] "GET /index.php/svg/core/filetypes/location?color=cc0000 HTTP/1.1" 404 5 "https://cloud.domain.tld/index.php/apps/maps/" "Mozilla/5.0 (xxxxxxxxxxxxxxx) Gecko/20100101 Firefox/112.0"
172.xxx.xxx.xxx - - [23/Apr/2023:20:47:23 +0000] "GET /index.php/svg/core/filetypes/location?color=00cc00 HTTP/1.1" 404 5 "https://cloud.domain.tld/index.php/apps/maps/" "Mozilla/5.0 (xxxxxxxxxxxxxxx) Gecko/20100101 Firefox/112.0"
172.xxx.xxx.xxx - - [23/Apr/2023:20:47:23 +0000] "GET /index.php/svg/core/actions/share?color=000000 HTTP/1.1" 404 5 "https://cloud.domain.tld/index.php/apps/maps/" "Mozilla/5.0 (xxxxxxxxxxxxxxx) Gecko/20100101 Firefox/112.0"
172.xxx.xxx.xxx - - [23/Apr/2023:20:47:23 +0000] "GET /index.php/svg/core/filetypes/location?color=0000cc HTTP/1.1" 404 5 "https://cloud.domain.tld/index.php/apps/maps/" "Mozilla/5.0 (xxxxxxxxxxxxxxx) Gecko/20100101 Firefox/112.0"
172.xxx.xxx.xxx - - [23/Apr/2023:20:47:23 +0000] "GET /index.php/svg/core/actions/starred?color=000000 HTTP/1.1" 404 5 "https://cloud.domain.tld/index.php/apps/maps/" "Mozilla/5.0 (xxxxxxxxxxxxxxx) Gecko/20100101 Firefox/112.0"
172.xxx.xxx.xxx - - [23/Apr/2023:20:47:23 +0000] "GET /index.php/svg/core/places/picture?color=000000 HTTP/1.1" 404 5 "https://cloud.domain.tld/index.php/apps/maps/" "Mozilla/5.0 (xxxxxxxxxxxxxxx) Gecko/20100101 Firefox/112.0"
```

my nginx config is almost identical to [nextcloud docs: Nextcloud in the webroot of NGINX](https://docs.nextcloud.com/server/latest/admin_manual/installation/nginx.html#nextcloud-in-the-webroot-of-nginx)

**Comment by alaskanpuffin** [comment]
This is resolved in #1084

**Comment by tacruc** [comment]
@alaskanpuffin feel free to close it directly the next time.
